From: Michael Biebl <biebl@debian.org>
Date: Thu, 18 Jul 2013 20:49:14 +0200
Subject: Launch logind via a shell wrapper

We need to setup the cgroupfs mount and the runtime directory before
starting logind. Use a shell script which is triggered by the D-Bus
service.
---
 src/login/org.freedesktop.login1.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/login/org.freedesktop.login1.service b/src/login/org.freedesktop.login1.service
index d10f631..82c007e 100644
--- a/src/login/org.freedesktop.login1.service
+++ b/src/login/org.freedesktop.login1.service
@@ -7,6 +7,6 @@
 
 [D-BUS Service]
 Name=org.freedesktop.login1
-Exec=/lib/systemd/systemd-logind
+Exec=/lib/systemd/systemd-logind-launch
 User=root
 SystemdService=dbus-org.freedesktop.login1.service
